src/git.js

```diff
@@ -366,15 +390,15 @@ export default class Git {
 	async push() {
 		if (FORK) {
 			return execCmd(
-				`git push -u fork ${ this.prBranch } --force`,
+				`git push -u fork ${ this.prBranch } --force-with-lease`,
```

Comment:
`--force-with-lease` adds an extra layer of security for preventing overwrite of changes when using the `git push` instead of the github API.